Prod 9232: fuel ui nics for fuel 10

rkhozinov opened the pull request.

Commit:
tox: ui env has been separated to 9 and 10 envs

Commit:
tox: ui env for update has been separated to 9 and 10 envs

Commit:
common: refacotoring to support 10 and 9 ui

Commit:
ui: 10 ui has been added

naumvd95 commented on servers/partner-ci/10.0/ui.yaml:
```diff
@@ -0,0 +1,232 @@
+    builders:
+      - get-props
+      - get-ui-refs
+#     - get-updates
```
why u disable this?
update master is true as seems above

rkhozinov replied:
variables from  the **get-updates** macros aren't needed for fuel10

naumvd95 commented on servers/partner-ci/common.yaml:
```diff
@@ -967,6 +1002,16 @@
             - "*.log"
           dirmatch: true
 
+- property:
```
why dont u use common property

rkhozinov replied:
jobs blocking isn't used for ui

rkhozinov committed:
Merge branch 'master' into PROD-9232

Commit:
refactor downloading-iso job

Commit:
refactoring prepare job-file, fix ui-job bugs

Commit:
code-style fixes

naumvd95 committed:
refactoring common-prepare code

naumvd95 committed:
code-style fixex, description added

rkhozinov committed:
Merge branch 'master' into PROD-9232

naumvd95 committed:
fix to avoid null-overwrite

Commit:
common: huge common refactoring

Commit:
common: huge common refactoring

Commit:
ui: 10 fuel ui rf related changes

naumvd95 committed:
Merge branch 'common.prepare.rf' of github.com:rkhozinov/jenkins-jobs into common.prepare.rf

Conflicts:
	servers/partner-ci/common.yaml

naumvd95 committed:
some bugfix

code-style fix

rkhozinov committed:
Merge pull request #180 from rkhozinov/common.prepare.rf-mk1

code-style fix for huge common rf

rkhozinov committed:
Merge pull request #179 from rkhozinov/common.prepare.rf

common: huge common refactoring

naumvd95 committed:
global merging

naumvd95 committed:
fix to avoid travis checks

naumvd95 committed:
fix incorrect logic

all necessary methods are already in "recreate_venv" function

rkhozinov committed:
Merge branch 'master' into PROD-9232

naumvd95 committed:
builders/:refactor tests.sh for jobs to associate with new macroses in common

common: join 2 macroses in 1

naumvd95 committed:
common: change injecting-case

common: previously-specific properties now named as macros-builder name
common: remove ";" delimeter from pure injecting case
common: inject-case bugfix

naumvd95 committed:
common: refactor file-writing cases

naumvd95 committed:
common:remove useless defining var

rkhozinov committed:
Merge pull request #171 from rkhozinov/PROD-9232-fix

ui-testing

naumvd95 committed:
10.0: job pure-deploy 10.0 fuel with 2 tests(smoke/bvt) (#186)

* create 10 pure fuel installation job



* 10.0/pure: fuel job

- remove snap-reverting case to better testing
10.0: 10,0 pure job with 2 tests (smoke/bvt)

rkhozinov committed:
Merge branch 'master' into PROD-9232
